docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3.8'
+
+services:
+  rusty_backend:
+    build: .
+    container_name: rusty_backend
+    working_dir: /rust
+    environment:
```

Comment:
Since we've already included the necessary Docker environment variables (`DOCKER_HOST`, `DOCKER_TLS_VERIFY`, and `DOCKER_CERT_PATH`) in our `Dockerfile`, you don't need to define them again in the `docker-compose.yml`. The environment variables in the `Dockerfile` will be set when the container starts.